docs/gallery/domain/plot_icephys_pandas.py

```diff
@@ -121,53 +118,54 @@
     print("%s (%s, %s) ----> %s" %
           ("    " * i, t['source_table'].name, t['source_column'].name, t['target_table'].name))
 
-#######################
+#####################################################################
 # Converting ICEphys metadata tables to pandas DataFrames
 # -------------------------------------------------------
 #
 
-#######################
+#####################################################################
 # Using nested DataFrames
 # ^^^^^^^^^^^^^^^^^^^^^^^
 # Using the :py:meth:`~hdmf.common.table.DynamicTable.to_dataframe` method we can easily convert tables
-# to pandas  `DataFrame <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html>`_.
+# to pandas `DataFrames <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html>`_.
```

Comment:
@rly note sphinx reuses names of links, i.e., when using `` `DataFrame`_  `` later on, sphinx will create the same link. Changing the link name here to ``DataFrames`` means the link will need to be redefined again later in order to avoid broken links. No big deal, I just wanted to let you know, since this is a bit obscure behavior of Sphinx.

Reply:
Ah, right. Good catch.